src/app/board-view/board-main/form-delete/form-delete.component.ts

```diff
@@ -14,5 +14,5 @@ export class FormDeleteComponent {
   }
   delete() {
     return this.delEv.emit(true);
-  }
+  } 
```

Comment:
canCelEv -  is not informative. It better to use the names of methods or property that clarify what it does ( full words). That may help you or your colleagues to read and understand your code